lib/src/rules/use_late_for_private_fields_and_variables.dart

```diff
@@ -111,6 +112,9 @@ class _Visitor extends UnifyingAstVisitor<void> {
       element = DartTypeUtilities.getCanonicalElementFromIdentifier(node);
     }
 
+    bool isNonNullable(DartType? type) =>
```

Comment:
Seems like this would be useful to have in the utilities library.

Reply:
Good idea!